src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,174 @@
+    }
+
+    private void printLines() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Tells the user that he has entered in valid query for the specified type
+     *
+     * @param taskType string, the type of task
+     */
+    public void showError(String taskType) {
+        this.printLines();
+        System.out.println("☹ OOPS!!! The description of a " + taskType + " cannot be empty.");
+        this.printLines();
+    }
+
+    /**
+     * Tells the user the number of tasks left
+     *
+     * @param i the number of tasks left
+     */
+    public void showNumTasks(int i) {
+        this.printLines();
+        System.out.println("Now you have " + i + " tasks in the list.");
+        this.printLines();
+    }
+
+    /**
+     * Sends the user a welcome message
+     */
+    public void hello() {
+        String logo =
+                "___________  __________  __________  ||   \n"
+                        + "|         | |         | |         |  ||      \n"
+                        + "-----------  ---------- ----------   ||  \n"
+                        + "    ||         ||          ||        ||       \n"
+                        + "    ||         ||          ||        ||           \n"
+                        + "    ||         ||          ||        ||           \n"
+                        + "    ||         ||          ||        ||       \n"
+                        + "    ||      __________  __________   ||      \n"
+                        + "    ||      |         | |         |  ______ \n"
+                        + "    ||      ----------  ----------   ______    \n";
+        this.printLines();
+        System.out.println("Hello! I'm \n" + logo);
+        System.out.println("What can I do for you?");
+        this.printLines();
+    }
+
+    /**
+     * says Goodbye to the user
+     */
+    public void goodbye() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    /**
+     * Notifies the user the tasks in list and the number of tasks
+     *
+     * @param tasks the TaskList of tasks
+     */
+    public void tasksInList(TaskList tasks) {
+        this.printLines();
+        System.out.println("Here are the tasks in your list:");
+        int i = 1;
+        for (Task t : tasks.getList()) {
+            System.out.println(i + "." + t.toString());
+            i++;
+        }
+        this.printLines();
+    }
+
+    /**
+     * Notifies the user that a task has been marked as done
+     *
+     * @param item the item marked as done
+     */
+    public void taskDone(Task item) {
+        this.printLines();
+        System.out.println("Nice! I've marked this task as done:");
+        System.out.println(item.toString());
+        this.printLines();
+    }
+
+    /**
+     * Notifies the user that a task has been unmarked
+     *
+     * @param item the item that has been unmarked
+     */
+    public void taskUndone(Task item) {
```

Comment:
The naming of this method can be more specific to suggest that it is a method to print the messages. Same for taskAdd and taskDelete